src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -19,11 +22,37 @@
 ;; return functions that accept a ring request map
 
 (defn command-app
-  "Function validating the request then submitting a command"
-  [version]
-  (-> enqueue-command
-    mid/verify-accepts-json
-    mid/verify-checksum
-    (mid/validate-query-params {:optional ["checksum"]})
-    mid/payload-to-body-string
-    (mid/verify-content-type ["application/json"])))
+  [{:keys [authorizer] :as globals}]
+  (let [{{:keys [connection endpoint]} :command-mq} globals
```

Comment:
Either way you're going to end up needing another line to get the connection and endpoint out of globals, but I can squish command-mq into the above line